pages/edit-tree.html

```diff
@@ -74,14 +85,20 @@
 
 	<div id="editProfileDiv" >
 		<div id="editProfileDivheader">
-			<img src="../images/cancel-icon.png" id="cancelEdit-icon" onclick='showDiv("editProfileDiv")' >
+			<img src="../images/cancel-icon.png" id="cancelEditIcon" onclick='showDiv("editProfileDiv")' >
 		</div>
 		<input type="file"  accept="image/*" name="image" id="file"  onchange="loadFile(event)" style="display: none;">
 		<label for="file" style="cursor: pointer;"><img src="" id="editUserPhoto"  /></label>
 		<input type="text" id="editName" placeholder="Name" class="editFactInput" > <br>
 		<input type="text" id="editSurname" placeholder="Surname" class="editFactInput" > <br>
 		<input type="text" id="editBirthDate" placeholder="Day of birth" class="editFactInput" > <br>
 		<input type="text" id="editDeathDay" placeholder="Death day" class="editFactInput" > <br>
+		<input type="text" id="editFather" placeholder="Father" class="editFactInput" > <br>
+		<input type="text" id="editMother" placeholder="Mother" class="editFactInput" > <br>
+		<input type="text" id="editNationality" placeholder="Nationality" class="editFactInput" > <br>
+		<input type="text" id="editSex" placeholder="Sex" class="editFactInput" > <br>
+		<input type="text" id="editBirthPlace" placeholder="Birth place" class="editFactInput" > <br>
+		<input type="text" id="editDeathCause" placeholder="Death cause" class="editFactInput" > <br>
 		<img src="../images/edit-icon.png" alt="" id="editInfo" onclick='editInfo()'>
 
```

Comment:
To wszystko powinno być formą i zawierać atrybuty name odpowiadające modelowi w api